File: trunk/C#/PokerWorld/Game/TrainingPokerGame.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Threading;

namespace PokerWorld.Game
{
    public class TrainingPokerGame : PokerGame
    {
        public TrainingTableInfo TrainingTable
        {
            get { return (TrainingTableInfo)m_Table; }
        }
        public TrainingPokerGame()
            : base()
        {
        }
        public TrainingPokerGame(AbstractDealer dealer) :
            base(dealer)
        {
        }

        public TrainingPokerGame(TrainingTableInfo table, int wtaPlayerAction, int wtaBoardDealed, int wtaPotWon)
            : base( table,  wtaPlayerAction,  wtaBoardDealed,  wtaPotWon)
        {
        }

        public TrainingPokerGame(AbstractDealer dealer, TrainingTableInfo table, int wtaPlayerAction, int wtaBoardDealed, int wtaPotWon)
            : base(dealer,table, wtaPlayerAction, wtaBoardDealed, wtaPotWon)
        {
        }
    }
}

File: JAVA/PokerWorld/src/bluffinmuffin/poker/TrainingPokerGame.java
package bluffinmuffin.poker;

import bluffinmuffin.poker.entities.TrainingTableInfo;
import bluffinmuffin.poker.entities.dealer.AbstractDealer;

public class TrainingPokerGame extends PokerGame implements IPokerGame
{
    public TrainingPokerGame()
    {
        super();
    }
    
    public TrainingPokerGame(TrainingTableInfo table, int wtaPlayerAction, int wtaBoardDealed, int wtaPotWon)
    {
        super(table, wtaPlayerAction, wtaBoardDealed, wtaPotWon);
    }
    
    public TrainingPokerGame(AbstractDealer dealer)
    {
        super(dealer);
    }
    
    public TrainingPokerGame(AbstractDealer dealer, TrainingTableInfo table, int wtaPlayerAction, int wtaBoardDealed, int wtaPotWon)
    {
        super(dealer, table, wtaPlayerAction, wtaBoardDealed, wtaPotWon);
    }
    
    public TrainingTableInfo getTrainingTable()
    {
        return (TrainingTableInfo) m_table;
    }
}